.github/workflows/main.yml

```diff
@@ -14,8 +14,7 @@ jobs:
       matrix:
         docker:
           - 'registry.fedoraproject.org/fedora:latest'
-          # Temporarily disabled as Fedora is moving to Python 3.10 and it is broken now
-          # - 'registry.fedoraproject.org/fedora:rawhide'
+          - 'fedora:rawhide'
```

Comment:
Why this instead of `'registry.fedoraproject.org/fedora:rawhide'`?

Reply:
I tested that previously, but it leads to the mentioned error.